include/fdreadoutlibs/wibeth/tpg/ProcessAVX2.hpp

```diff
@@ -20,23 +20,21 @@
 
 namespace swtpg_wibeth {
 
-
 template<size_t NREGISTERS>
 inline void
 process_window_avx2(ProcessingInfo<NREGISTERS>& info)
 {
-  //const __m256i adcMax = _mm256_set1_epi16(info.adcMax);
+  const __m256i overflowMax = _mm256_set1_epi16(INT16_MAX);
 
   // Pointer to keep track of where we'll write the next output hit
   __m256i* output_loc = (__m256i*)(info.output); // NOLINT(readability/casting)
 
-  const __m256i iota = _mm256_set_epi16(15, 14, 13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 0);
+  const __m256i iota = _mm256_set_epi16(14, 13, 12, 11, 10, 9, 8, 15, 7, 6, 5, 4, 3, 2, 1, 0);
```

Comment:
Thanks for catching this! This was fixed properly in the test application used for exercising precisely the correct unpacking but somehow it got lost when moving it into the production code. I suggest to add it also to the ProcessRSAVX2 code or make an issue to not forget about it.

Reply:
Hi Adam, ok, thanks for noticing the channel sequence in ProcessRSAVX2. Instead of creating an issue, it was a quick fix in aabedabu/adc_peak_feature, which is now updated for this PR.